services/lab-service/lab/Makefile

```diff
@@ -0,0 +1,14 @@
+all: build
+
+build:
+	tox -e pep8,py35
+	.tox/py35/bin/python setup.py bdist_wheel
```

Comment:
remove wheel and use raw setup.py install, also your command install app into virtual env don't do that.